README.md

```diff
@@ -27,15 +48,30 @@ The standalone engine that powers the multiplayer first person shooter [Unvanqui
 
 ### Linux, Mac OS X, MSYS
 
-  1. `mkdir build && cd build`
-  2. `cmake ..`
-  3. `make`¹
+Produces files will be stored in a new directory named `build`.
```

Comment:
This sentence is a bit off, s/Produces/The resulting/ for example.

Reply:
Hmm, I wanted to say _**Produced** files_